lang/cs/Org.Apache.REEF.Tests/Functional/Telemetry/MetricsDriver.cs

```diff
@@ -134,7 +135,7 @@ public void OnError(Exception error)
         /// </summary>
         private void UpdateMetrics(TestSystemState systemState)
         {
-            var driverMetrics = new DriverMetrics(systemState.ToString(), DateTime.Now);
+            var driverMetrics = new DriverMetrics(EventPrefix + systemState.ToString(), DateTime.Now);
```

Comment:
No need to call `.ToString()` explicitly. Simply do `EventPrefix + systemState`

Reply:
Done